guides/common/modules/proc_preparing-a-host-for-external-databases.adoc

```diff
@@ -24,8 +24,14 @@ ifndef::orcharhino[]
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # subscription-manager repos --disable '*'
+ifdef::satellite[]
 # subscription-manager repos --enable={RepoRHEL7ServerSoftwareCollections} \
 --enable={RepoRHEL7Server} --enable={RepoRHEL7ServerSatelliteServerProductVersion}
+endif::[]
+ifndef::satellite[]
+# subscription-manager repos --enable={RepoRHEL7ServerSoftwareCollections} \
+--enable={RepoRHEL7Server}
+endif::[]
```

Comment:
Unsure if this is correct, because you left out the "if not satellite" version in "guides/common/modules/proc_creating-a-local-file-repository.adoc". Looks inconsistent.

Reply:
"It's complicated" ;-)

Right now, it renders with "--enable={RepoRHEL7ServerSatelliteServerProductVersion}" for all builds (besides Orcharino), but that repo is only really available if you're running Satellite. The introduced `ifdef` omits that one repo for Katello builds.

In reality, Katello users then need to add the Katello repo that contains the "PostgreSQL EVR" extension (which is why we enable the Satellite repo on Satellite builds), but this omission was already present before my change and I wanted to first focus on getting things building w/o missing attributes.

So, I'd argue the if/else here is as correct as it is (or can be, w/o re-doing half of that doc, which I am not interested in today).

As for `proc_creating-a-local-file-repository.adoc` there we only enable the Satellite Tools repo, which is not available outside of Satellite. Here, again, the doc would need an addition where to get that package *instead* in the non-Satellite case, but it was also already "broken" before the added `ifdef`.

Reply:
OK, understood. Then I'd kindly request to use "if def foreman-el,katello" instead.

Reply:
(done)

Reply:
Much appreciated. We use this procedure downstream & I saw that you intend to cherry-pick to 3.1 :)